packages/components/psammead-social-embed/src/__snapshots__/index.test.jsx.snap

```diff
@@ -59,17 +116,34 @@ exports[`AmpSocialEmbed should render a notice when the provider is unsupported
 <div
   class="c0"
 >
-  <p>
-    Sorry but we're having trouble displaying this content
-  </p>
   <a
-    href="embed-url"
+    class="c1"
+    href="#skip-unknown-content"
   >
-    View content on unknown
+    Skip unknown content
   </a>
-  <small>
-    Warning: BBC is not responsible for third party content
-  </small>
+  <div
+    class="c2"
+  >
+    <p>
+      Sorry but we're having trouble displaying this content
+    </p>
+    <a
+      href="embed-url"
+    >
+      View content on unknown
+    </a>
+    <small>
+      Warning: BBC is not responsible for third party content
+    </small>
+  </div>
+  <p
+    class="c3"
+    id="skip-unknown-content"
+    tabindex="-1"
+  >
+    End of unknown content
```

Comment:
Is this the text we are actually using? (Translated)

Reply:
No, this is test data.

Reply:
Could you give me an example please? "End of Twitter content"?

Reply:
@lizcameron has kindly reminded me what this will say on the fallback, same as the enriched embed - Thanks both